app/views/planning_applications/show.html.erb

```diff
@@ -1,108 +1,111 @@
 <%= render "assessment_dashboard" do %>
   <% if @planning_application.in_progress? %>
-    <h2 class="app-task-list__section">
-      <span class="app-task-list__section-number">Process Application</span>
+    <h2 class="govuk-heading-l">
+      Validation
     </h2>
-    <div class="govuk-summary-list">
-      <div class="govuk-summary-list__row">
-        <ol class="app-task-list">
-          <li>
-            <h2 class="app-task-list__section">
-              <span class="app-task-list__section-number">1. </span> Validation
-            </h2>
-          </li>
-        </ol>
-        <ul class="app-task-list__items" id="validation-section">
-          <li class="app-task-list__item">
-            <span class="app-task-list__task-name">
-              <% if @planning_application.can_validate? %>
-                <%= link_to "Validate application", validate_form_planning_application_path(@planning_application), aria: { describedby: "validation-completed" }, class: "govuk-link" %>
-              <% else %>
-                Validate application
-              <% end %>
-            </span>
-            <% if @planning_application.validation_complete? %>
-              <%= tag.strong "Completed", id: "validation-completed", class: "govuk-tag app-task-list__task-tag" %>
+    <ul class="app-task-list__items" id="validation-section">
+      <li class="app-task-list__item">
+        <span class="app-task-list__task-name">
+          <% if @planning_application.can_validate? %>
+            <%= link_to "Validate application", validate_form_planning_application_path(@planning_application), aria: { describedby: "validation-completed" }, class: "govuk-link" %>
+          <% else %>
+            Validate application
+          <% end %>
+        </span>
+        <% if @planning_application.validation_complete? %>
+          <%= tag.strong "Completed", id: "validation-completed", class: "govuk-tag app-task-list__task-tag" %>
+        <% end %>
+      </li>
+    </ul>
+    <h2 class="govuk-heading-l">Assess against policy</h2>
+    <ul class="app-task-list__items" id="assess-policy-section">
+      <% @planning_application.policy_classes.each do |klass| %>
+        <li class="app-task-list__item">
+          <span class="app-task-list__task-name">
+            <%= link_to klass, {
+                controller: "policy_assessments",
```

Comment:
I'm finding the use of "klass" everywhere really confusing. Is it necessary here? Would it not be easier to read if we used  "policy" or"policy_class" for the iteration? 

It feels like"klass" is everywhere and its making the code a bit tricky, not quite as straightforward as it could be with plainer english